rclcpp/test/rclcpp/test_copy_all_parameter_values.cpp

```diff
@@ -86,3 +86,19 @@ TEST_F(TestNode, TestParamCopyingExceptions)
   node2->declare_parameter("Foo1", rclcpp::ParameterValue(0.123));
   EXPECT_NO_THROW(rclcpp::copy_all_parameter_values(node1, node2, override));
 }
+
+TEST_F(TestNode, TestFileImport)
+{
+  const uint64_t expected_param_count = 4;
+  auto load_node = std::make_shared<rclcpp::Node>(
```

Comment:
we can add `LifecycleNode` test case here?

Reply:
Would that not make a circular testing structure, as `LifecycleNode` in `lifecycle_rclcpp` needs to include `rclcpp`.